kore/src/Kore/Strategies/Goal.hs

```diff
@@ -225,11 +225,9 @@ onePathFollowupStep claims axioms =
         , Simplify
         , TriviallyValid
         , DeriveSeq claims
-        , RemoveDestination
         , Simplify
         , TriviallyValid
         , DeriveSeq axioms
-        , RemoveDestination
```

Comment:
I think that this change makes sense, but I still wonder why we added the `RemoveDestination` thing to `deriveSeq` in the first place (my previous PR moved it from `deriveSeq` here). Anyway, could you also remove one of the `RemoveDestination` calls from the first step above?

Reply:
done